### PR TITLE
Download issues about SSIS extension

### PR DESCRIPTION
**Pull request by WendyYes**

### CODE REVIEW

**Comment by PRMerger9** [comment]
@WendyYes : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by chugugrace — APPROVED** [review]

File: docs/ssdt/release-notes-ssdt-2019.md [hunk 1]
```diff
@@ -59,3 +56,18 @@ More information is available in release notes.**
...
+If you get an error during installation, you can check the logs under %temp%\SsdtisSetup.
+Usually, the detail error log is at the end of Microsoft.DataTools.IntegrationServices_{timstamp}_ISVsix.log. 
+ - If the error is "The file {filefullpath} already exists." 
+    1. set idepath=”C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7”  (replace with your path)
+	    rm %idepath%\IDE\CommonExtensions\Microsoft\SSIS\* 
+	    rm %idepath%\IDE\PublicAssemblies\SSIS\* 
+	    rm %idepath%\IDE\PublicAssemblies\Microsoft BI\Business Intelligence Projects\Integration Services\*
+	  2. Repair the vs2019 
+    3. Restart and start ssdt setup install again.
+ - If the error is "Microsoft.VisualStudio.Setup.CanceledByPrecheckException: Pre-check verification failed with warning(s) :  AnotherInstallationRunning."
```
Comment (Jak-MS) [inline docs/ssdt/release-notes-ssdt-2019.md:70]:
```suggestion
If you get an error during installation, you can check the logs under %temp%\SsdtisSetup.
Usually, the detail error log is at the end of Microsoft.DataTools.IntegrationServices_{timstamp}_ISVsix.log. 
- If the error is "The file {filefullpath} already exists." 
   1. set idepath=”C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7”  (replace with your path)
      ```
      	    rm %idepath%\IDE\CommonExtensions\Microsoft\SSIS\* 
	    rm %idepath%\IDE\PublicAssemblies\SSIS\* 
	    rm %idepath%\IDE\PublicAssemblies\Microsoft BI\Business Intelligence Projects\Integration Services\*
      ```
   2. Repair the vs2019 
   3. Restart and start ssdt setup install again
- If the error is "Microsoft.VisualStudio.Setup.CanceledByPrecheckException: Pre-check verification failed with warning(s) :  AnotherInstallationRunning."
```

**Comment by PRMerger3** [comment]
@Jak-MS : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/ssdt/release-notes-ssdt-2019.md [hunk 2]
```diff
@@ -59,3 +56,20 @@ More information is available in release notes.**
...
+ 
+If you get an error during installation, you can check the logs under %temp%\SsdtisSetup.
+Usually, the detail error log is at the end of Microsoft.DataTools.IntegrationServices_{timstamp}_ISVsix.log. 
+- If the error is "The file {filefullpath} already exists." 
+   1. set idepath=”C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7”  (replace with your path)
+      ```
+      	    rm %idepath%\IDE\CommonExtensions\Microsoft\SSIS\* 
+	    rm %idepath%\IDE\PublicAssemblies\SSIS\* 
+	    rm %idepath%\IDE\PublicAssemblies\Microsoft BI\Business Intelligence Projects\Integration Services\*
+      ```
```
Comment (Jak-MS) [inline docs/ssdt/release-notes-ssdt-2019.md:69]:
```suggestion
      ```
      rm %idepath%\IDE\CommonExtensions\Microsoft\SSIS\* 
      rm %idepath%\IDE\PublicAssemblies\SSIS\* 
      rm %idepath%\IDE\PublicAssemblies\Microsoft BI\Business Intelligence Projects\Integration Services\*
      ```
```

**Comment by PRMerger3** [comment]
@Jak-MS : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**